quantulum3/_lang/en_US/tests/extract_spellout_values.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+
+from ..parser import extract_spellout_values
+
+TEST_CASES = [
+    ("one hundred and five", ["105.0"]),
+    ("a million", ["1000000.0"]),
+    ("a million and one", ["1000001.0"]),
+    ("million", ["1000000.0"]),
+    ("million and one", ["1000001.0"]),
+    ("one hundred million", ["100000000.0"]),
+    ("one hundred and five million", ["105000000.0"]),
+    ("half", ["0.5"]),
+    ("two and a half", ["2.5"]),
+    ("two and a half million", ["2500000.0"]),
+    ("twenty six million and seventy two hundred", ["26007200.0"]),
+    ("twenty", ["20.0"]),
+    ("zero", ["0.0"]),
+    ("several hundred years", []),
+    ("Zero is a small number.", ["0.0", "1.0"]),
+    # ("a million and a half", ["1500000.0"]), # this is a hard one
```

Comment:
Wolframalpha parses this as 1000000.5 which I find reasonable. 150000 would be one and a half million.
![image](https://user-images.githubusercontent.com/20638630/210150957-973af1b9-ea85-4115-8573-bb808aa0efa1.png)